R/linear-mixed-model-development.R

```diff
@@ -262,7 +259,7 @@ LinearMixedModelDevelopment <- R6Class("LinearMixedModelDevelopment",
       } else if (self$params$type == 'regression') {
         private$fitLogit <- glm(
           as.formula(paste(self$params$predictedCol, '.', sep = " ~ ")),
-          data = private$dfTrain,
+          data = private$dfTrainRaw,
```

Comment:
Unless there's a compelling reason to feed dummified data into model fitting, I'd leave it alone...